xcat3/plugins/control/openbmc.py

```diff
@@ -0,0 +1,94 @@
+from oslo_log import log
+from xcat3.plugins.control import base
+from xcat3.common import exception
+from xcat3.common import states
+from xcat3.common import rest
+
+import json
+import pdb
+
+LOG = log.getLogger(__name__)
+
+url_dict  = {"set_power_state" : "/xyz/openbmc_project/state/host0/attr/RequestedHostTransition",
+             "get_power_state"  : "/xyz/openbmc_project/state/host0" }
+
+data_dict = {states.POWER_COMMAND_ON   : "xyz.openbmc_project.State.Host.Transition.On",
+             states.POWER_COMMAND_OFF  : "xyz.openbmc_project.State.Host.Transition.Off",
+             states.POWER_COMMAND_RESET: "xyz.openbmc_project.State.Host.Transition.Reboot"}
+
+class OPENBMCPlugin(base.ControlInterface):
+
+    def validate(self, node):
+        bmc = node.control_info.get('bmc')
```

Comment:
Seems this is copy from ipmi.py,   can you unify the name,  it is `bmc` here but `bmc_address` at ipmi.py ?